XpressAI/xircuits
# ✨Render python component descriptions as markdown

# Description

This will render component's description as markdown. Also, can use HTML style. (Not recommended)

![Markdown in description v2](https://user-images.githubusercontent.com/84708008/178920118-6707f1a3-ee93-4b36-b99d-e4b1dc65adae.gif)

## Pull Request Type


## Type of Change


# Tests

1. Use markdown style on component's description
   1. Open any node's script
   2. Edit its docstring as markdown (As shown in gif)
   3. Open node's description on canvas
   4. It should style accordingly
   5. And any links should be in blue color and clickable

## Tested on?


# Notes

Can also use HTML style. (Not recommended)
